src/pages/Designer.js

```diff
@@ -745,24 +745,34 @@ const Designer = ({ update, viewMode = false, sharedMetadata = undefined }) => {
 
     const { parentId, id } = useParams();
     const [projectParentId, setProjectParentId] = useState(parentId || location.state?.parentId);
-
     const [projectName, setProjectName] = useState(null);
 
     const loadData = async () => {
         if (initialized && parentId && id) {
             try {
-                const response = await fetch(process.env.REACT_APP_API_BASE_URL + '/blueprints/' + id, {
-                    method: 'get',
-                    headers: {
-                        'Content-Type': 'application/json',
-                        Authorization: initialized ? `Bearer ${keycloak?.token}` : undefined,
-                    },
-                });
-
+                var response;
+                if (parentId === 'admin') {
+                    response = await fetch(process.env.REACT_APP_API_BASE_URL + '/api/refArchs/' + id, {
+                        method: 'get',
+                        headers: {
+                            'Content-Type': 'application/json',
+                            Authorization: initialized ? `Bearer ${keycloak?.token}` : undefined,
+                        },
+                    });
+                } else {
+                    response = await fetch(process.env.REACT_APP_API_BASE_URL + '/blueprints/' + id, {
+                        method: 'get',
+                        headers: {
+                            'Content-Type': 'application/json',
+                            Authorization: initialized ? `Bearer ${keycloak?.token}` : undefined,
+                        },
+                    });
+                }
                 if (response.ok) {
                     const result = await response.json();
                     if (result?.metadata) {
-                        setProjectParentId(result.parentId);
+                        if (parentId === 'admin') setProjectParentId(parentId);
```

Comment:
if and else should be syntaxed as 
if { 
.. 
} else  {
...
}

Improves readability!